### PR TITLE
Preview form parameter, with tests

### PR DESCRIPTION
**Pull request by wpride**
@benrudolph amazingly, this will work. Hold off merge until I can do the JS side

### CODE REVIEW

File: src/main/java/application/MenuController.java [hunk 1]
```diff
@@ -83,6 +82,11 @@ public Object navigateSessionWithAuth(@RequestBody SessionNavigationBean session
                 menuSession.getSessionWrapper().syncState();
             } else {
                 menuSession = performInstall(sessionNavigationBean, authToken);
+                // If we have a preview command, load that up
+                if(sessionNavigationBean.getPreviewCommand() != null){
+                    menuSession.getSessionWrapper().setCommand(sessionNavigationBean.getPreviewCommand());
```
Comment (benrudolph) [inline src/main/java/application/MenuController.java]: i have no idea how this works, but awesome!!

**Comment by benrudolph** [comment]
nice, this is huge

**Comment by wpride** [comment]
@benrudolph we should pair on this today

**Comment by benrudolph** [comment]
👍

**Comment by wpride** [comment]
cross https://github.com/dimagi/commcare-hq/pull/12925

File: src/main/java/application/MenuController.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import auth.HqAuth;
 import beans.InstallRequestBean;
 import beans.NotificationMessageBean;
+import beans.PreviewRequestBean;
```
Comment (benrudolph) [inline src/main/java/application/MenuController.java]: did you want to remove this before i merge?

**Comment by benrudolph** [comment]
(deletion coming in follow up PR)

**Pull request merged**